packages/sanity/src/core/form/inputs/arrays/ArrayOfObjectsInput/List/PreviewItem.tsx

```diff
@@ -67,14 +67,11 @@ export function PreviewItem<Item extends ObjectItem = ObjectItem>(props: Preview
   } = props
   const {t} = useTranslation()
 
-  const treeEditing = useTreeEditingEnabled()
-  const treeEditingDisabledByOption = parentSchemaType?.options?.treeEditing === false
-  const legacyEditing = treeEditingDisabledByOption || treeEditing.legacyEditing
+  const {enabled, legacyEditing} = useTreeEditingEnabled()
 
   // The edit portal should open if the item is open and:
   // - tree array editing is disabled
-  // - legacy array editing is enabled (e.g. in a Portable Text editor)
-  const openPortal = open && (!treeEditing.enabled || legacyEditing)
+  const openPortal = open && (!enabled || legacyEditing)
```

Comment:
Removing this since we don't want to provide this right now (or even in the future, but if we decide we want to we can return)